src/server/schema.js

```diff
@@ -146,12 +146,16 @@ const getAggregationType = (entry) => {
   return '';
 };
 
-const getAggregationSchemaForOneIndex = (esInstance, esIndex, esType) => {
+const getAggregationSchemaForOneIndex = (esInstance, esDict) => {
+  const esIndex = esDict.index;
+  const esType = esDict.type;
+  const histogramTypePrefix = 'RegularAccess';
```

Comment:
you can move this `histogramTypePrefix` const even out of each individual function to save more